tests/test_benchmark.py

```diff
@@ -487,7 +476,7 @@ def test_memops(self):
         return 0;
       }
     '''
-    self.do_benchmark('memops', src, 'final:', emcc_args=['-s', 'MINIMAL_RUNTIME=1'])
+    self.do_benchmark('memops', src, 'final:', emcc_args=['-s', 'MINIMAL_RUNTIME=0'])
```

Comment:
Why this?

Reply:
MINIMAL_RUNTIME doesn't work with the wasm backend yet. I could remove this entirely, but this does mark the tests that are ready for that mode (i.e. work in that mode with asm2wasm, and hopefully soon with upstream too).